packages/app/src/systems/Account/pages/Logout/Logout.tsx

```diff
@@ -42,12 +78,30 @@ export const Logout = () => {
           <Text color="gray11" css={{ mb: '$2' }}></Text>
         </Card>
       </Dialog.Description>
+      <ControlledField
+        control={control}
+        name="logoutConfirmation"
+        label={
+          <p>
+            Write <i>{logOutConfirmationPhrase}</i> to logout
+          </p>
+        }
+        render={({ field }) => (
+          <Input>
+            <Input.Field
+              {...field}
+              aria-label="Confirm Log Out"
+              placeholder={logOutConfirmationPhrase}
+            />
+          </Input>
+        )}
+      />
```

Comment:
why is the controlled field outside of <Dialog.Description>
if you put this in the right place, layout will look ok already

Reply:
thanks